monitoring/alerts/vgalerts.libsonnet

```diff
@@ -0,0 +1,28 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'vg-alert.rules',
+        rules: [
+          {
+            alert: 'VolumeGroupUsageAtThreshold',
+            expr: |||
+              (topolvm_volumegroup_size_bytes - topolvm_volumegroup_available_bytes)/topolvm_volumegroup_size_bytes > %(vgUsageThreshold)0.2f
+            ||| % $._config,
+            'for': $._config.volumegroupUsageThresholdAlertTime,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: |||
+                'VolumeGroup usage has crossed the provided threshold of %(vgUsageThreshold)0.2f'
```

Comment:
Please add the volume group and node name as well. I believe it is available in the topolvm metrics :

```curl -s 127.0.0.1:41685/metrics | grep -Ei 'topolvm_volumegroup_.*?_bytes\{'
topolvm_volumegroup_available_bytes{device_class="vg1",node="kube-node"} 4.790222323712e+12
topolvm_volumegroup_size_bytes{device_class="vg1",node="kube-node"} 4.800959741952e+12```

Reply:
Apologies for the dumb question @nbalacha - but why are the labels called device_class and node? the content appears to be a vg name, the metric name is a volumegroup..and does the node refer to a hostname? If the labels are aligned to the schema in node-exporter for example, you can use promql more easily to merge the content.

Reply:
@nbalacha done

Reply:
@pcuzner , Not dumb at all. These metrics are exposed by the topolvm csi driver and they use something called a deviceClass  which internally maps to the VG. The labels are from topolvm.